contrib/kind.sh

```diff
@@ -1309,6 +1309,7 @@ function is_nested_virt_enabled() {
 }
 
 function install_kubevirt() {
+        local kubevirt_version="v1.0.0"
```

Comment:
while I agree this makes sense (IMHO tracking nightly builds was too aggressive) can you elaborate why you're doing this *now* ? 

And if you don't mind, move that into its own commit.